src/collections/_documentation/platforms/dotnet/wpf.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Windows Presentation Foundation
+sidebar_order: 40
+---
+
+Sentry's .NET SDK works with Windows Presentation Foundation applications through the [Sentry NuGet package](https://www.nuget.org/packages/Sentry). It works with WPF apps running on .NET Framework 4.6.1, .NET Core 3.0 or higher.
+
+## Configuration
+
+Besides the [main configuration like calling `SentrySdk.Init`](/platforms/dotnet/), you must consider the WPF specific configuration.
+
+The SDK automatically captures unhandled exceptions that hit the `AppDomain.UnhandledException`, but on WPF, for production apps (when no debugger is attached), WPF catches exception to show the dialog to the user. You can also configure your app to capture those exceptions before the dialog shows up:
```

Comment:
suggest: "The SDK automatically handles AppDomain.UnhandledException. On WPF, for production apps (when no debugger is attached), WPF captures an exception to show dialog to the user."